<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navbar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLabel" name="navbar_background">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>1321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: lightgrey;
border: 1px solid grey;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="navbar_profile">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: grey;
border-radius: 25px;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="nav_env">
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
border-radius: 10px;
text-align: center;
font-weight: bold; height: auto;</string>
        </property>
        <property name="text">
         <string>Environments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nav_model_list">
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
border-radius: 10px;
text-align: center;
font-weight: bold; height: auto;</string>
        </property>
        <property name="text">
         <string>Your Models</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nav_model_create">
        <property name="styleSheet">
         <string notr="true">border: 1px solid grey;
border-radius: 10px;
text-align: center;
font-weight: bold; height: auto;</string>
        </property>
        <property name="text">
         <string>Create Models</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="loading" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>851</height>
     </rect>
    </property>
    <widget class="QLabel" name="logo_cont">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>1311</width>
       <height>861</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content_window" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>90</y>
      <width>1301</width>
      <height>761</height>
     </rect>
    </property>
    <widget class="QWidget" name="env_window" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1301</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>1221</width>
        <height>581</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1219</width>
         <height>579</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1221</width>
          <height>581</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="1" column="2">
          <widget class="QLabel" name="env6">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="env5">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="env2">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="env1">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="env4">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="env3">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="env7">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="env8">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="env9">
           <property name="styleSheet">
            <string notr="true">margin: 10px;
background: lightgrey;
border-radius: 30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="model_list_window" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>1301</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <widget class="QLineEdit" name="searchbar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey; border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_model_button">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
border-radius: 20px;
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_model_button">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
border-radius: 20px;
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_model_button">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
border-radius: 20px;
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="model_list_table">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>1071</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="content_background">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>5</y>
       <width>1311</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="model_creation_window" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>1301</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <zorder>env_window</zorder>
    <zorder>content_background</zorder>
    <zorder>model_creation_window</zorder>
    <zorder>model_list_window</zorder>
   </widget>
   <zorder>loading</zorder>
   <zorder>navbar</zorder>
   <zorder>content_window</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
